src/legacy/ui/ui_render/ui_render_mixin.js

```diff
@@ -221,6 +221,19 @@ export function uiRenderMixin(kbnServer, server, config) {
       ...kbnServer.newPlatform.setup.plugins.uiPlugins.public.entries()
     ].map(([id, plugin]) => ({ id, plugin }));
 
+    const injectedVars = await replaceInjectedVars(
+      request,
+      mergeVariables(
+        injectedVarsOverrides,
+        await server.getInjectedUiAppVars(app.getId()),
+        defaultInjectedVars,
+      ),
+    );
+
+    // TODO: move uiCapabilities to uiExport
+    // Deleting this for testing to verify that no consuming code is trying to read this value.
+    delete injectedVars.uiCapabilities;
```

Comment:
I plan to move `uiCapabilities` to be defined as a UI Export. Now that we're not reading this data as an `injectedVar` and we're no longer using the `replaceInjectedVars` machinery, I don't see any benefit to leaving these definitions in the `injectedVar`s at all.